### PR TITLE
I became the GOATiest GOAT tonight

### PR DESCRIPTION
**Pull request by blakeparkinson**
DONT MERGE:

1. I duplicated the function fetchLeadersLists into the quiz route b/c i just wanted to see if it would work.
2. i dont think we can extend routes to other routes. Maybe we make another model to avoid duplicate function. NOTE: MAKE A NEW MODEL

3. The data is getting to the render. know i just need to figure out what to display so that it matches with out current front end.

4. Im the :goat:

### CODE REVIEW

**Comment by blakeparkinson** [comment]
TODO #5: There is too much data being pumped to the leagesearch page, and it is slow. The Data that goes there needs to be references and not the whole platter of stuff.

Update the importer script to also create these reference documents

**Comment by blakeparkinson** [comment]
@kjgross kulfee. This is ready to review. You made a real bad boog with the top scores insert so i fixed that in the script. It should be working end to end now.

File: scripts/team_import.js [hunk 1]
```diff
@@ -23,10 +23,19 @@ var endpoint = '';
...
 //process.argv grabs the command line arguments
 var league = process.argv[2];
 
+//deleting a team sample
+/*var d = {};
+d.type = 'leaders';
+
+teams_model.deleteTeam(d);
+return;*/
+
```
Comment (kjgross) [inline scripts/team_import.js]: delete ^^. you have it in teams model

File: routes/quiz.js [hunk 2]
```diff
@@ -17,9 +18,22 @@ router.get('/', function(req, res) {
...
         api_team_id = items.api_team_id;
         quiz_name = items.quiz_name;
+        type = items.type;
         if (list_id){ 
             players_model.fetchGoatPlayers(list_id, rb_team_id, league, res, req, players_model.returnPlayers)
           }
+        else if (type == 'leaders'){
+          leaders_model.fetchLeadersLists(league, function(doc){
+            res.render('quiz', {
+              clock: '2:00',
```
Comment (kjgross) [inline routes/quiz.js]: define this at the top as a var?

File: routes/teams.js [hunk 3]
```diff
@@ -32,27 +32,44 @@ router.get('/quiz', function(req, res) {
...
-        var api_team_id = item.team_id;
-        var quiz_name = item.market + ' ' + item.name;
-        createQuiz(rb_team_id, list_id, league, quiz_name, res, returnItem, api_team_id);
-      }
-      else{
-        res.status(500);
-        res.json('error', { error: 'unable to find a matching team' });
-      }
-    });
+    if (type != undefined){
```
Comment (kjgross) [inline routes/teams.js]: Make this type == leaders    so that we don't have to come back and change it in two days when we add a type somewhere else?

**Comment by kjgross** [comment]
What was my booog? That I didn't clear that array each time after inserting?

This looks :peach: -y  nice job!

File: scripts/team_import.js (hunk 1 above)
Comment (blakeparkinson) [inline scripts/team_import.js]: ok

File: routes/teams.js (hunk 3 above)
Comment (blakeparkinson) [inline routes/teams.js]: ok

File: routes/quiz.js (hunk 2 above)
Comment (blakeparkinson) [inline routes/quiz.js]: ok

**Pull request merged**